src/compiler/transformers/static-to-meta/component.ts

```diff
@@ -65,7 +65,7 @@ export const parseStaticComponentMeta = (
     componentClassName: cmpNode.name ? cmpNode.name.text : '',
     elementRef: parseStaticElementRef(staticMembers),
     encapsulation,
-    shadowDelegatesFocus: parseStaticShadowDelegatesFocus(encapsulation, staticMembers),
+    shadowDelegatesFocus: !!parseStaticShadowDelegatesFocus(encapsulation, staticMembers),
```

Comment:
this necessitates an edit to a spec file (`src/compiler/transformers/test/parse-encapsulation.spec.ts`) but I believe this is safe to do because the only reference to this field is here:

https://github.com/ionic-team/stencil/blob/1c359016e09b66aeb82c704a934d1355cceb4cb0/src/utils/format-component-runtime-meta.ts#L26-L28

we're checking that the value is truthy here, so coercing `null` to `false` won't have any effect on runtime behavior.

_however_ the `parseStaticShadowDelegatesFocus` function does specifically return `null` if the encapsulation mode isn't shadow:

https://github.com/ionic-team/stencil/blob/1c359016e09b66aeb82c704a934d1355cceb4cb0/src/compiler/transformers/static-to-meta/encapsulation.ts#L29-L48

however given that `shadowDelegatesFocus` property here is typed `boolean` and we're not doing anything with that `null` return value here I think coercing here is the right thing to do (although we could also make the return type for `parseStaticShadowDelegatesFocus` always be `boolean`, which would have the same result overall)

Reply:
I think that makes sense!